app/services/unpublish_service.rb

```diff
@@ -6,8 +6,15 @@ def withdraw(edition, public_explanation, user = nil)
       edition.document.lock!
       check_unpublishable(edition)
 
-
-      withdrawal = Withdrawal.new(public_explanation: public_explanation)
+      previous_withdrawal = edition.withdrawn? && edition.status.details
```

Comment:
This is similar to what @1pretz1 has been doing with the 'published_status'. I wonder if it would be simpler to just `dup` any existing withdrawal, to make it clear that we're just changing the explanation?

Reply:
Yup, dup sounds good.